specification/paths/MultiColliShipments.json

```diff
@@ -38,12 +46,6 @@
                     }
                   }
                 },
-                "label_mime_type": {
-                  "type": "string",
-                  "description": "Deprecated, use `meta.label.mime_type` instead. The requested mime-type for this shipment's label.",
-                  "example": "application/pdf",
-                  "deprecated": true
-                },
```

Comment:
Why are you removing these? These were added in addition to the properties in the meta.label property, so that we could have a seemless integration for new microservices.

Reply:
None of the microservices have implemented the multi-colli endpoit yet, so I thought let's not introduce deprecated properties here.